.devcontainer/Dockerfile

```diff
@@ -1,75 +1,57 @@
-#-------------------------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
-#-------------------------------------------------------------------------------------------------------------
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.137.0/containers/go/.devcontainer/base.Dockerfile
+ARG VARIANT="1.15"
```

Comment:
Wondering if building this is going to slow things down significantly? Do you know how long it takes to build?

Reply:
it takes about 3 minutes (can see in the Checks below); the new validation takes 12 minutes end-to-end compared to 17 minutes for the current version

Reply:
Nice, if it's faster that's good. Might get even faster if you bypass the linting on the apis directory again (I think you dropped that before as I mentioned above)

Reply:
fixed that, wasn't actually invoking lint on CI at all :embarrassing_face: